api/dbadapters/bigquery.ts

```diff
@@ -199,13 +200,68 @@ export class BigQueryDbAdapter implements IDbAdapter {
       `SELECT * FROM ${CACHED_STATE_TABLE_NAME}`,
       5000 // TODO: Add pagination for 5000+ rows
     );
-    const persistedMetadata = rows.map(row => {
-      const encodedProto = Buffer.from(row.proto, "base64");
+    const persistedMetadata = rows.map((row: IMetadataRow) => {
```

Comment:
i feel like we should put this code, plus the `IMetadataRow` interface, inside some common utility location.